examples/utils/example_node.rs

```diff
@@ -81,7 +81,7 @@ impl ExampleNode {
                 }
                 Event::RestartRequired => {
                     info!("{} Received RestartRequired event", self.get_debug_name());
-                    self.node = unwrap_result!(Node::builder().create(self.sender.clone()));
+                    let new_node = unwrap!(Node::builder().create(self.sender.clone()));
```

Comment:
I've tried to keep the merged changes to just the unwrap!s, but I must have missed a few (especially those large blocks... I'm embarrassed by those :p). Thanks!